internal/server/server.go

```diff
@@ -7,25 +7,19 @@ import (
 	"log"
 	"net/http"
 
-	"github.com/ayeniblessing101/recipe-book/internal/models"
+	"github.com/ayeniblessing101/recipe-book/internal/handlers"
 	"github.com/davecgh/go-spew/spew"
 )
 
 func handleHelloWorld(w http.ResponseWriter, r *http.Request) {
-	recipe := &models.Recipe{
-		ID: 23,
-		Name: "Singaporean Noodles",
-		Ingredients: []string{"Rice Noodle", "Red Bell Pepper", "Green Bell Pepper", "Onion", "Garlic"},
-		Directions: []string{"Boil Water", "Soak Noodles in Hot Water", "Fry Vegetables"},
-		Calories: 23,
-	}
 	fmt.Fprintf(w, "%s\n", "Hello Dima")
-	fmt.Fprintf(w, html.EscapeString(spew.Sdump(recipe)))
-
+	fmt.Fprintf(w, html.EscapeString(spew.Sdump()))
 }
 
 // Server method handles all requests
 func Server(port string) {
 	http.HandleFunc("/", handleHelloWorld)
+	http.HandleFunc("/categories", handlers.AddCategory)
+	http.HandleFunc("/categories", handlers.GetCategories)
```

Comment:
Attach handlers to different paths, otherwise they will overwrite each other.

Reply:
Ok... Thanks Dima